pkg/xsd/element.go

```diff
@@ -22,13 +22,13 @@ type Element struct {
 	refElm          *Element     `xml:"-"`
 	ComplexType     *ComplexType `xml:"complexType"`
 	SimpleType      *SimpleType  `xml:"simpleType"`
-	schema          *Schema      `xml:"-"`
+	Schema          *Schema      `xml:"-"`
```

Comment:
this is exported so it can be accessed in the template. i can try to clean that up if needed. i think it was so the included template could access the namespace of the whole XSD.